fe/src/pages/Login/Login.module.scss

```diff
@@ -0,0 +1,32 @@
+.input {
```

Comment:
We have **Chakra UI** for main css building, we should only use _scss_ when we absolutely have to.